wcf.targets

```diff
@@ -47,4 +47,9 @@
      <WcfToolsOutputPath>$(WcfRootFolder)bin\Wcf\tools\</WcfToolsOutputPath>
   </PropertyGroup>
 
+  <!-- Override ValidateIgnoreReference from Target 'ValidatePackageTargetFramework' in PackageLibs.targets -->
+  <ItemGroup>
+    <ValidateIgnoreReference Include="System.Private.ServiceModel" />
+  </ItemGroup>
+
```

Comment:
What error where you getting before you did this?  I'd expect ValidatePackageTargetFramework to walk the graph to S.P.SM and evaluate all of its references to determine if you are setting the right generation.